src/objects/api/filters.py

```diff
@@ -1,15 +1,67 @@
+from django.utils.translation import ugettext_lazy as _
+
 from django_filters import filters
 from vng_api_common.filtersets import FilterSet
 from vng_api_common.utils import get_help_text
 
 from objects.core.models import Object
 
+from .constants import Operators
+from .utils import display_choice_values_for_help_text, is_number
+from .validators import validate_data_attrs
+
 
 class ObjectFilterSet(FilterSet):
     type = filters.CharFilter(
         field_name="object_type", help_text=get_help_text("core.Object", "object_type")
     )
+    data_attrs = filters.CharFilter(
```

Comment:
I think this is a pretty smart way to do it but i differs alot with regular API query params.

I would prefer typical `key=value` pairs. Can we perhaps prefix query params for the `records.data` with `records__data__` so it looks like normal query params? Something like:

`?record__data__height__exact=100` and `?record__data__dimensions__height__exact=100`

Reply:
Moved to #90